03/app.js

```diff
@@ -1,14 +1,40 @@
 console.log('DOM');
 
-const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
+const classElement = document.querySelector ('.parent-for-button');
+if(classElement) {
+    const btn = document.createElement( 'button');
+
+    const buttonSettings = {
+        attr: {
+            className: 'btn',
+            title: 'super button'
+        },
+        css: {
+            border: '1px solid #336699',
+            padding: '5px 20px',
+            color: '#444'
+        },
+        text: 'Click me!',
+    }
+    
+    for (let newObject in buttonSettings) {
```

Comment:
Lepiej nazwać zmienną przez `key`, `prop`, bo to nie jest `newObject`.
Ponieważ ta zmienna nie jest nadpisywana to lepiej użyć `const`.